pytorch_tabnet/tab_model.py

```diff
@@ -132,6 +132,7 @@ def update_fit_params(
                   "Use reshape(-1, 1) for single regression."
             raise ValueError(msg)
         self.output_dim = y_train.shape[1]
+        self.preds_mapper = None
```

Comment:
Why do we reset the preds_mapper?

If one is already given, isn't it a bit dangerous to recreate one? What if the mapping is different from the first training, re-training is going to be messed up completely no?

Reply:
This is only in the case of Regression, where previously `preds_mapper` was not defined. Since we're saving it, I preferred to to this instead of an existence check.